### ISSUE TITLE
KeyedVectors.get_embedding_layer(train_embeddings = False) returns a trainable Embedding layer.

### ISSUE THREAD

**Issue by vrmpx** [RaRe-Technologies/gensim#1557]
#### Description
`KeyedVectors.get_embedding_layer` does not set `trainable` when returning the Keras Embedding layer.

#### Steps/Code/Corpus to Reproduce
```
from gensim.models.keyedvectors import KeyedVectors

embeddings = KeyedVectors.load_word2vec_format("GoogleNews-vectors-negative300.bin", binary = True)
embedding_layer = embeddings.get_embedding_layer(train_embeddings=False)
print(embedding_layer.trainable)
```

#### Expected Results
False

#### Actual Results
True

#### Versions
Linux-4.4.0-93-generic-x86_64-with-debian-stretch-sid
Python 3.6.1 |Continuum Analytics, Inc.| (default, May 11 2017, 13:09:58)
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]
NumPy 1.13.1
SciPy 0.19.1
gensim 2.3.0
FAST_VERSION 2

**Comment by zsef123** [comment]
[keyedvectors.py](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/keyedvectors.py) : 835

`layer = Embedding(input_dim=weights.shape[0], output_dim=weights.shape[1], weights=[weights])`

`get_embedding_layer` function is not use `train_embeddings` parameter

**Comment by menshikh-iv** [comment]
Fixed in #1558
